scripts/rabbitmq-env

```diff
@@ -62,8 +62,11 @@ RABBITMQ_HOME="$(rmq_realpath "${RABBITMQ_SCRIPTS_DIR}/..")"
 ## Set defaults
 . ${RABBITMQ_SCRIPTS_DIR}/rabbitmq-defaults
 
+DEFAULT_SCHEDULING_BIND_TYPE_FLAG="tnnps"
+[ "x" = "x$RABBITMQ_SCHEDULING_BIND_TYPE_FLAG" ] && RABBITMQ_SCHEDULING_BIND_TYPE_FLAG=${DEFAULT_SCHEDULING_BIND_TYPE_FLAG}
+
 ## Common defaults
-SERVER_ERL_ARGS="+P 1048576"
+SERVER_ERL_ARGS="+P 1048576 +stbt $RABBITMQ_SCHEDULING_BIND_TYPE_FLAG "
```

Comment:
I'd rename this to `RABBITMQ_SCHEDULER_BIND_TYPE`. That's how the docs describe this option (whether that's the clearest name from the English grammar point of view). I don't think we use the `_FLAG` prefix for other values.